### DISCUSSION TITLE
Release 2.0.2

### DISCUSSION THREAD

**Discussion by gallandYoann** [dompdf/dompdf#3022] [General]
Hello, 

Do you have a date estimate for release 2.0.2 ?

Thank's

**Comment by bsweeney** [comment]
Not at this time. There's some documentation that still needs to be finalized for previous releases (2.0.0 and 2.0.1). Is there a particular update you want to see in that release? At some point I'll pare down the issues I plan to address.

**Reply by gallandYoann** [reply]
Hi, thank you for your answer, ok I understand.
Yes i wait this issue : https://github.com/dompdf/dompdf/issues/1388
But are you going to develop a fix or just merge the associated PR? Because i test the fix and if i have a one rowspan on two page this fix don't work

Thanks
Have a good day

**Reply by bsweeney** [reply]
As you noted the referencing PR doesn't fully address the issue. So, though we will consider merging that PR for a future release that would not close the issue itself. Meaning we will look towards further changes specific to 1388.

I can't say as of now if that will be addressed with 2.0.2. The current milestone is basically the wishlist of issues to address. As we work through the milestone some issues may be pushed to a future release. So for now my suggestion is to just subscribe to that issue so you can receive updates on its progress.

**Reply by gallandYoann** [reply]
Ok thanks
